mailboxes/lang/de

```diff
@@ -1,21 +1,22 @@
-index_title=Lese Benutzer-Mail
-index_none=Sie dürfen die Mails der Benutzer auf diesem System nicht lesen.
+index_title=Benutzer-Mail lesen
+index_none=Die Mails der Benutzer dürfen nicht gelesen werden.
```

Comment:
In English, we originally say, "You are not allowed to read email for any users on this system".

Now, I'm curious why this was changed to a more impersonal form in your German variant. Why did you choose "Die Mails der Benutzer dürfen nicht gelesen werden" instead of using a more direct version, as in the original English variant?

Don't we want to directly and specifically address the user in this case?

Reply:
> Noted.

Vielen Dank!

> What do you prefer then?

That I don't know! It's up to you to decide!

> For native speakers, that's a massive difference in both, reading and hearing, and shouldn't be generalised.

Yeah, then I agree that using `Sie` is better for the German language!

> You either want to offer two language file options (like WordPress does with de_de and de_de_informal) or make it a rule of thumb throughout all translations to either use informal or formal language

No need for extra complexity. I think just using the formal form would be better.